### DISCUSSION TITLE
[v3] Lifecycle hooks: rendering and rendered don't work

### DISCUSSION THREAD

**Discussion by N1ebieski** [livewire/livewire#6265] [7. Bugs (archived)]
### Livewire version

v3.0.0-beta.7

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Create a Component Class with content:

```php
<?php

namespace App\Livewire\Components;

use Livewire\Component;

class Hello extends Component
{
    public function rendered($view, $html)
    {
        dd('Livewire is great but bugged a lot!');
    }    

    public function render()
    {
        return <<<'HTML'
            <div>
                <h1>Hello!</h1>
            </div>
        HTML;
    }
}
```

2. A rendered() method will not initiate

### Additional information

In Livewire\Features\SupportLifecycleHooks\SupportLifecycleHooks class you forgot to add hooks. Should be:

```php
public function render($view, $data)
{
    $this->callHook('rendering', ['view' => $view, 'data' => $data]);
    $this->callTraitHook('rendering', ['view' => $view, 'data' => $data]);

    return function ($html) use ($view) {
        $this->callHook('rendered', ['view' => $view, 'html' => $html]);
        $this->callTraitHook('rendered', ['view' => $view, 'html' => $html]);
    };
}
```

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@N1ebieski thanks for reporting! Any chance you could submit a PR with a test and the changes suggested? Thanks!
